### PR TITLE
Create journal-of-neuroimaging.csl

### LINKED ISSUE #6047: Style Request: Journal of Neuroimaging
Hi, I tried editing the Neurology CSL style (closest fit) but I have some strange aberrations (shows author affiliations in bibliography and uses year as edition). Appreciate your help.

A link to online style documentation: https://onlinelibrary.wiley.com/page/journal/15526569/homepage/forauthors.html 

For journals, the journal's ISSN (print version) and/or e-ISSN (online version): 1051-2284

Two citations, for a journal article and a book chapter, in the format of the style you're requesting:
In-text citation (superscript):
1
2

Bibliography:
1. Campbell JL, Pedersen O K. The varieties of capitalism and hybrid success. Comp Political Stud 2007; 40: 307–32. 
2. Mares I. Firms and the welfare state: When, why, and how does social policy matter to employers? In Hall PA & Soskice D, ed. Varieties of capitalism. The institutional foundations of comparative advantage. New York: Oxford University Press; 2001: 184–213

### PR DESCRIPTION
**Pull request by POBrien333**
closes #6047

### CODE REVIEW

File: journal-of-neuroimaging.csl [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                  <group>
+                    <date variable="issued">
+                      <date-part name="month" suffix=" "/>
+                      <date-part name="day" suffix=", "/>
+                      <date-part name="year"/>
+                    </date>
+                  </group>
+                  <text variable="page"/>
+                </group>
+              </else>
```
[the remark this answers is not in the document]
Comment (POBrien333) [reply journal-of-neuroimaging.csl:157]: Thanks for reviewing, but I'm not sure what you mean?
I ran it through formatter.citationstyles.org/ and it's not making a change.

[the remark this answers is not in the document]
Comment (adam3smith) [reply journal-of-neuroimaging.csl:157]: yeah, indenting looks fine, I'm guessing Brenton got thrown by the various levels of else/else-if

[the remark this answers is not in the document]
Comment (adam3smith) [reply journal-of-neuroimaging.csl:157]: Also, the CI automatically fixes indenting if it makes any changes to the style

**Comment by adam3smith** [comment]
Thanks!

**Comment by POBrien333** [comment]
Instead of a comment here, can you put this into an issue here? 
https://github.com/citation-style-language/styles/issues

**Pull request merged**